scripts/aws/cfn/microservice.yml

```diff
@@ -58,6 +58,14 @@ Parameters:
     Type: String
     Description: 'ARN of dynamodb table containing notifications'
 
+  NotificationsCostDynamoTableName:
+    Type: String
+    Description: 'Notifications Cost Dynamo table name'
+
+    NotificationsCostDynamoTableArn:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Parameter NotificationsCostDynamoTableName has invalid property NotificationsCostDynamoTableArn